docs/core/rid-catalog.md

```diff
@@ -119,7 +119,7 @@ wish to check if new ones are added, please check back here.
     * `linuxmint.17.2-x64`
     * `linuxmint.17.3-x64`
 
-## OS X RIDs
+## macOS RIDs
```

Comment:
This really is an OS X RID. The version change starts with Sierra. El Capitan and before are still OS X. It's a question of how we're going to show the two sets of RIDs once Sierra is out. I'd suggest that we list them as two separate categories.